.travis.yml

```diff
@@ -14,7 +14,7 @@ sudo: false  # use the new container-based Travis infrastructure
 dist: trusty # Ubuntu 14.04
 
 mono:
-  - latest
+  - nightly
```

Comment:
Yep, I'm pretty sure we could get it "running" with the latest stable (if you can call it like this, we always had some failures on travis because of mono bugs, even before updating to 5).
But you have to setup msbuild by hand, I don't think it's worth the effort.

When nightly "works" and is sometimes red it's actually as good as ever ;)